core/src/test/scala/akka/persistence/r2dbc/journal/PersistTimestampSpec.scala

```diff
@@ -48,12 +50,12 @@ class PersistTimestampSpec
       (1 to 100).foreach { n =>
         val p = n % numberOfEntities
         // mix some persist 1 and persist 3 events
+        val event = s"e$p-$n"
```

Comment:
ok, so an ordinary String without any json formatting can also be stored in `JSONB`, and that's why all tests can be run in that mode?

Reply:
No, the test will fail if you use the JSON or JSONB schema while running the tests (that's why I made all the test data be valid JSON in my original request). I have been planning on adding separate tests that run using JSON data stored in a JSON schema, but haven't gotten around to it yet unfortunately. I will try to put something together before the end of the summer.

Reply:
I see. I thought it just worked because you still had the ci workflow with ` -Dakka.persistence.r2dbc.journal.payload-column-type=jsonb`. Makes sense to have a separate test.